File: IndexSynchronizer/wwwroot/js/index-sync/ConnectionDetails.js
class ConnectionDetails {
    constructor(username, password, server, database, schemaname, tablename, isSource) {
        this.Username = username.toString();
        this.Password = password.toString();
        this.ServerName = server.toString();
        this.DatabaseName = database.toString();
        this.SchemaName = schemaname.toString();
        this.TableName = tablename.toString();
    }
}

File: IndexSynchronizerServices/Models/ConnectionDetails.cs
namespace IndexSynchronizerServices.Models
{
    public class ConnectionDetails : IConnectionDetails
    {
        public ConnectionDetails() { }
        public String Username { get; set; }
        public String Password { get; set; }
        public String ServerName { get; set; }
        public String DatabaseName { get; set; }
        public String SchemaName { get; set; }
        public String TableName { get; set; }
    }
}
